src/ui/EmojiReactions/ReactionItem.tsx

```diff
@@ -49,6 +55,27 @@ export default function ReactionItem({
     && (channel?.isGroupChannel() && !channel.isSuper);
 
   const handleOnClick = () => {
+    if (isFiltered && !reactedByMe) {
+      openModal({
```

Comment:
@git-babel If you don't mind, could you post a screenshot of this case?

Reply:
![Uploading 스크린샷 2024-08-29 오후 4.26.12.png…]()

Reply:
<img width="1289" alt="스크린샷 2024-08-29 오후 4 26 12" src="https://github.com/user-attachments/assets/34544504-3dd6-4178-9d95-181f2b0a6349">